build.gradle

```diff
@@ -58,6 +58,7 @@ repositories {
 
 final htsjdkVersion = System.getProperty('htsjdk.version','2.14.1')
 final picardVersion = System.getProperty('picard.version','2.17.2')
+final barclayVersion = System.getProperty('barclay.version','2.0.0-1-g9417229-SNAPSHOT')
```

Comment:
I assume that this will be changed to a release version before merge of this branch?

Reply:
Yeah - I did mention that in the PR comments ;-). Its in a separate commit for that reason - now that the rest of the tests are squared away I'll start the Barclay release and update this to use the released version.